packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -121,3 +124,33 @@ class ScrollCompletedSemanticsEvent extends SemanticsEvent {
     return map;
   }
 }
+
+/// An event for a semantic announcement.
+///
+/// This should be used for announcement that are not seamlessly announced by
```

Comment:
Might be worth pointing out that this should be used very sparingly as advised by most a11y guidelines as these announcements can be fairly disruptive.